src/ui/popup.js

```diff
@@ -534,6 +534,7 @@ export default class Popup extends Evented {
             this._closeButton = DOM.create('button', 'mapboxgl-popup-close-button', this._content);
             this._closeButton.type = 'button';
             this._closeButton.setAttribute('aria-label', 'Close popup');
+            this._closeButton.setAttribute('aria-hidden', '');
```

Comment:
Any reason not to set explicitly to `true` as the ticket suggests?
```suggestion
            this._closeButton.setAttribute('aria-hidden', 'true');
```

Reply:
Maybe set to `'aria-hidden'` or otherwise add a comment? `// The empty string is the recommended truthy value for setAttribute`. Seems a bit surprising/confusing, though there's relatively low maintenance burden here.

Reply:
> I looked up the setAttribute docs and it recommends using an empty string in the example: https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute#example

That example is using the `disabled` attribute, which is a [boolean attribute](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attribute), thus the empty string.

But for [`aria-hidden`](https://www.w3.org/TR/wai-aria/#aria-hidden), the undefined (default) value is described as:

> _The element's hidden state is determined by the user agent based on whether it is rendered._

and we do not want that, because we want the element to be hidden from accessibility APIs even when it is rendered. It should be set to `true`:

> _The element is hidden from the accessibility API._